fw/http.c

```diff
@@ -5402,6 +5402,11 @@ tfw_http_req_process(TfwConn *conn, TfwStream *stream, const TfwFsmData *data)
 	 */
 	req->cache_ctl.timestamp = tfw_current_timestamp();
 	req->jrxtstamp = jiffies;
+	/*
+	 * Bypass cache if corresponding binary flag in request set
+	 */
+	if (unlikely(test_bit(TFW_HTTP_B_CHAIN_NO_CACHE, req->flags)))
+		req->cache_ctl.flags |= TFW_HTTP_CC_CHAIN_NO_CACHE;
```

Comment:
I considered this. But there are downsides. Separate binary flag in request allow us fully match all rules, possible revert cache flag in case of multiple rule matches, for instance:

```
{
cookie "mycookie" == "nocache*" -> $cache = 0;
hdr "My-Header" == "docache" -> $cache = 1;
}
```

Reply:
Updated implementation to `TFW_HTTP_CC_NO_CACHE` flag.